### DISCUSSION TITLE
Timezone can't change before start container

### DISCUSSION THREAD

**Discussion by CatTimothy** [nextcloud/all-in-one#1857] [Questions] [answered]
### Steps to reproduce
1.Setup hostname
2.Change Timezone to TW
3.Start Container

### Expected behavior 
All Run Successful

### Actual behavior 
Nextcloud and Apache still running
Database logged [database "nextcloud_database" does not exist]

### Host OS 
window
#### Nextcloud AIO version 
v4.2.0
#### Current channel 
latest

**Comment by szaimen** [comment, marked as answer]
Hi, can you post the database container logs here?
